### PR TITLE
feat(DDS): add dds parameter template resource

### PR DESCRIPTION
**Pull request by houpeng80**
**What this PR does / why we need it**:
  add dds parameter template resource
**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
  add dds parameter template resource
```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud/services/acceptance/dds/' TESTARGS='-run TestAccDdsParameterTemplate_basic' 
==> Checking that code complies with gofmt requirements... 
TF_ACC=1 go test ./huaweicloud/services/acceptance/dds/ -v -run TestAccDdsParameterTemplate_basic -timeout 360m -parallel 4 
=== RUN   TestAccDdsParameterTemplate_basic 
=== PAUSE TestAccDdsParameterTemplate_basic
=== CONT  TestAccDdsParameterTemplate_basic
--- PASS: TestAccDdsParameterTemplate_basic (16.81s) 
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dds       16.862s
```

### CODE REVIEW

File: docs/resources/dds_parameter_template.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  + **mongos**: the mongos node type.
+  + **shard**: the shard node type.
+  + **config**: the config node type.
+  + **replica**: the replica node type.
+  + **single**: the single node type.
+
+  Changing this parameter will create a new resource.
+
+* `node_version` - (Required, String, ForceNew) Specifies the database version.
+  The value can be **4.4**, **4.2**, **4.0**, **3.4**, or **3.2**.
```
Comment (deer-hang) [inline docs/resources/dds_parameter_template.md]: remove the last `,` or `or`

File: docs/resources/dds_parameter_template.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+* `readonly` - Indicates whether the parameter is read-only.
+  + If the value is **true**, the parameter is read-only.
+  + If the value is **false**, the parameter is not read-only.
+
+## Import
+
+The DDS parameter template can be imported using the `id`, e.g.
+
+```
```
Comment (deer-hang) [inline docs/resources/dds_parameter_template.md]: last `bash`

File: docs/resources/dds_parameter_template.md (hunk 1 above)
Comment (houpeng80) [reply docs/resources/dds_parameter_template.md]: done

File: docs/resources/dds_parameter_template.md (hunk 2 above)
Comment (houpeng80) [reply docs/resources/dds_parameter_template.md]: done

**Pull request merged**